docker-compose.yml

```diff
@@ -33,38 +35,59 @@ services:
   ranker:
     build:
       context: .
-      dockerfile: docker/Dockerfile.ranker
+      dockerfile: Dockerfile
     depends_on:
       - redis
     environment:
       REDIS_CONNECTION_STRING: redis://redis:6379
       CELERY_BROKER: redis://redis-celery-broker:6380
       CELERY_BACKEND: redis://redis-celery-broker:6380
-    command: ["poetry", "run", "uvicorn", "--host", "0.0.0.0", "--port", "5001", "ranking_server:app"]
+    command: ["uvicorn", "--host", "0.0.0.0", "--port", "5001", "dante.app.ranking_server.ranking_server:app"]
     ports:
       - "5001:5001"
+    volumes:
+      - type: bind
+        source: ${PWD}/config.ini
```

Comment:
There seems to be a missing config.ini file? When I run `make run` it looks for an config.ini in the root project folder. 
---
Error response from daemon: invalid mount config for type "bind": bind source path does not exist: /host_mnt/Users/baott/osome-csdl-ranking-challenge/config.ini

Reply:
Yes you will need to supply a config.ini in the same folder where you run docker-compose, in this case the root of the repository. This is meant to supply the configuration without needing to rebuild the image, which will be helpful when deploying in production. But for now the contents of this file are just the same as the sample config file in the dante/osomerank. So you can just copy that there at the root of the repository. 

In the docker-compose file there is also another bind mount for the cache. This is also to keep the image size small and avoiding to rebuild images every time the cache is cleared. For that you will need to copy your `data/` and `models/` folder to your user cache dir as described in the "Breaking Changes" section above. Alternatively you can run `dante.osomerank.load_all()` once look into the logs the cache location there and then point the docker-compose file there. Just make sure to include your values in the `S3` section of the config file.